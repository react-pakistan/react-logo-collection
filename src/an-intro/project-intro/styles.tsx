import styled from 'styled-components';

export const ProjectIntroContainer = styled.div`
  display: flex;
  flex-flow: column nowrap;
  justify-content: center;
  align-items: center;
`;
