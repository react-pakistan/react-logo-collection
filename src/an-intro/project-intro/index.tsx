import React from 'react';
import { Image } from '@react-pakistan/react-ui-collection/eCommerceV1UI/image';
import { H3 } from '@react-pakistan/react-ui-collection/eCommerceV1UI/h3';
import { HR } from '@react-pakistan/react-ui-collection/eCommerceV1UI/hr';
import { H4 } from '@react-pakistan/react-ui-collection/eCommerceV1UI/h4';
import { ProjectIntroContainer } from './styles';

export const ProjectIntro = ({

} : IProjectIntroProps) : JSX.Element => (
  <ProjectIntroContainer>
    <Image
      imageUrl="https://i.ibb.co/k8Y3RMh/React-Pakistan-Logo-01.jpg"
      alt="alt-image"
      margin="0"
      padding="0"
      width="600px"
      height="auto"
    />
    <H3
      fontFamily="Montserrat"
      fontSize="1.6em"
      fontWeight={700}
      lineHeight="normal"
      letterSpacing="1px"
      textAlign="left"
      textTransform="uppercase"
      color="#2E2E2E"
    >
      This project is released under ReactPakistan platform.
    </H3>
    <HR />
    <H4
      fontFamily="Montserrat"
      fontSize="1.4em"
      fontWeight={700}
      lineHeight="normal"
      letterSpacing="1px"
      textAlign="center"
      textTransform="uppercase"
      color="#2E2E2E"
    >
      Project Maintainer's List
    </H4>
  </ProjectIntroContainer>
);

export interface IProjectIntroProps {
  /**
   *
   */
  test? : string;
}
