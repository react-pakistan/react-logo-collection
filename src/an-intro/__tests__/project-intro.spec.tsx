import React from 'react';
import { shallow } from 'enzyme';
import { ProjectIntro } from '../project-intro';
import { projectIntroProps } from '../__stories__/components/project-intro.story';

describe('Testing ProjectIntro component', () => {
  it('matches ProjectIntro snapshot', () => {
    const wrapper = shallow(<ProjectIntro {...projectIntroProps()} />);
    expect(wrapper).toMatchSnapshot();
  });
});
