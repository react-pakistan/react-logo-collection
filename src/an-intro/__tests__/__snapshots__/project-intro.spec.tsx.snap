// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing ProjectIntro component matches ProjectIntro snapshot 1`] = `ShallowWrapper {}`;
